platforms/android/library-compose/src/main/java/io/element/android/wysiwyg/compose/RichTextEditor.kt

```diff
@@ -120,9 +124,17 @@ private fun RealEditor(
                             state.linkAction = linkAction
                         }
                     addTextChangedListener {
+                        val prevInternalHtml = state.internalHtml
+                        val prevMarkdown = state.messageMarkdown
+
                         state.internalHtml = getInternalHtml()
                         state.messageHtml = getContentAsMessageHtml()
                         state.messageMarkdown = getMarkdown()
+
+                        // This is needed so we don't send a typing status when the editor is first displayed
+                        if (prevInternalHtml != state.internalHtml || prevMarkdown != state.messageMarkdown) {
+                            onTyping()
```

Comment:
Do we want to provide a boolean to indicate if the content is empty (so the app should consider that the user stops typing) of not (so the user starts/continues typing)? Or can this will be handled (and debounce/filtered) app side?

Reply:
Oh, I guess we could. The Rust code also has some automatic 4s timeout for sending a 'not typing anymore' event if I'm not mistaken, so these could complement each other.